README.md

```diff
@@ -10,31 +11,75 @@ This chart will deploy the following on the target OpenShift cluster:
 - ArgoCD `orchestrator` project (optional: disabled by default)
 - Tekton tasks and build pipeline (optional: disabled by default)
 
+## Important Note for ARM64 Architecture Users
+Note that as of November 6, 2023, OpenShift Serverless Operator is based on RHEL 8 images which are not supported on the ARM64 architecture. Consequently, deployment of this helm chart on an [OpenShift Local](https://www.redhat.com/sysadmin/install-openshift-local) cluster on MacBook laptops with M1/M2 chips is not supported.
+
 ## Prerequisites
-- You logged in to a Red Hat OpenShift Container Platform (version 4.13+) cluster as a cluster administrator.
+- Logged in to a Red Hat OpenShift Container Platform (version 4.13+) cluster as a cluster administrator.
 - [OpenShift CLI (oc)](https://docs.openshift.com/container-platform/4.13/cli_reference/openshift_cli/getting-started-cli.html) is installed.
 - [Operator Lifecycle Manager (OLM)](https://olm.operatorframework.io/docs/getting-started/) has been installed in your cluster.
 - Your cluster has a [default storage class](https://docs.openshift.com/container-platform/4.13/storage/container_storage_interface/persistent-storage-csi-sc-manage.html) provisioned.
 - [Helm](https://helm.sh/docs/intro/install/) v3.9+ is installed.
 - [PostgreSQL](https://www.postgresql.org/) database is available with credentials to manage the tablespace (optional).
-  - A [reference implementation](#postgresql-deployment-reference-implementation) is provided for your convenience.
-- A Github API Token - to import items into the catalog, there is a need for GITHUB_TOKEN with the permissions as detailed [here](https://backstage.io/docs/integrations/github/locations/). For classic token, include the following permissions: repo (all), admin:org (read:org) and user (read:user, user:email).
-- `ArgoCD/OpenShift GitOps` operator is installed and one instance of `ArgoCD` exists in a given namespace (later referenced by `ARGOCD_NAMESPACE` env var)
+  - A [reference implementation](https://github.com/parodos-dev/orchestrator-helm-chart#deploying-postgresql-reference-implementation) is provided for your convenience.
+- A GitHub API Token - to import items into the catalog, ensure you have a GITHUB_TOKEN with the necessary permissions as detailed [here](https://backstage.io/docs/integrations/github/locations/). For classic token, include the following permissions:
+  - repo (all)
+  - admin:org (read:org)
+  - user (read:user, user:email)
+
+## Installation
+
+### Quick installation
+1. Follow instruction [here](https://github.com/parodos-dev/orchestrator-helm-chart/blob/main/GitOps.md)
+2. Run the following command to set up environment variables:
+    ```console
+    ./hack/setenv.sh
+    ```
+  - Accept all default values
+  - Provide GH Token (created as describe in prerequisite section)
+3. Source the environment variables by running
+    ```console
+    source .env
+    ```
```

Comment:
Why 2 steps and not just 1?
The script can use the `GITHUB_TOKEN` already set (if present) to allow non-interactive mode
Similarly, the script could source the output file, so the user only has to run the script

Reply:
Also, the script is only needed if we want to deploy tekton and Argo CD, so those should already be installed (if we want to use them)

Reply:
I have a doubt here, are `--set rhdhOperator.k8s.clusterToken=$K8S_CLUSTER_TOKEN --set rhdhOperator.k8s.clusterUrl=$K8S_CLUSTER_URL` necessary only for argoCD/tekton?

Reply:
I think, yes, @dmartinol keep me true here

Reply:
They are a bit more generic and allow to fetch the workflow resources (deployment, pod) from the cluster. So they are partly independent from the ArgoCd/Tekton option